### DISCUSSION TITLE
please help, how to config the search css style in kbar?

### DISCUSSION THREAD

**Discussion by white0dew** [timlrx/pliny#180] [General]
please help, how to config the search css style in kbar?

**Comment by timlrx** [comment]
Currently there's limited customization option and probably the main thing you could do is to modify the styles with a tailwind style sheet since the component uses tailwind css tokens. If larger changes are required, feel free to just [copy the code](https://github.com/timlrx/pliny/blob/main/packages/pliny/src/search/KBarModal.tsx) and modify accordingly.

Here's an example of a tailwind config that affects the kbar style - https://github.com/timlrx/tailwind-nextjs-starter-blog/blob/main/tailwind.config.js. Mostly customising the primary color and gray color of choice.
